### ISSUE TITLE
Constructor completion

### ISSUE THREAD

**Issue by soyuka** [kristijanhusak/deoplete-phpactor#10]
Works very well, thanks for the plugin. I have an issue though when typing a constructor I don't have any completion.

**Comment by kristijanhusak** [comment]
Do you have it with omnifunc?

**Comment by soyuka** [comment]
insert mode <C-X><C-O> on my constructor shows the class indeed:
![2020-02-26-104544_1585x331_scrot](https://user-images.githubusercontent.com/1321971/75332960-90b69180-5885-11ea-8f08-afe0b57a6cd0.png)

**Comment by kristijanhusak** [comment]
What you mean by "insert mode" ?
When you set omnifunc like this `setlocal omnifunc=phpactor#Complete`, and trigger autocompletion with <C-x><C-o>, do you get this?

**Comment by soyuka** [comment]
Oh I missed some characters, let me try again. I do have `autocmd FileType php setlocal omnifunc=phpactor#Complete` in my configuration.

So, I do have the above completion, but these are the current available classes. When I type the parenthesis for example: `new Documentation(` the completion is empty. Would it be possible to have the constructor signature?

**Comment by kristijanhusak** [comment]
What you mean by signature? Deoplete offers only autocompletion, it doesn't do anything else.

**Comment by soyuka** [comment]
phpactor gives the signature as part of the autocompletion, for example if you use https://github.com/Shougo/echodoc.vim you'll get the signature after choosing a completion item (<C-Y>). 

Note that I don't have any completion with classes, and I need to check the constructor signature by hand with phpactor afterwards.

**Comment by kristijanhusak** [comment]
How do you check signature with phpactor, using `PhpactorHover` ?

if it shows signature when autocompleting other things, like methods or such, and it doesn't show for constructor, i would say this is not the issue of this plugin. I just forward data from phpactor rpc to deoplete, and when it has it, it shows it, but if it's not there, which is probably the case for constructor, it doesn't show anything.

**Comment by soyuka** [comment]
Indeed, anyway I'm now trying vim-lsp (https://github.com/prabirshrestha/vim-lsp/) and it works even better then phpactor.

**Comment by kristijanhusak** [comment]
With which LSP?

**Comment by soyuka** [comment]
mattn/vim-lsp-settings installed intelephense (https://www.npmjs.com/package/intelephense)
